bundles/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/internal/api/model/SystemConfig.java

```diff
@@ -15,29 +15,33 @@
 import java.util.List;
 
 /**
- * The {@link FreeboxSystemConfig} is the Java class used to map the "SystemConfig"
+ * The {@link SystemConfig} is the Java class used to map the "SystemConfig"
  * structure used by the system API
  * https://dev.freebox.fr/sdk/os/system/#
  *
  * @author Laurent Garnier - Initial contribution
  */
-public class FreeboxSystemConfig {
+public class SystemConfig {
     private String firmwareVersion;
     private String mac;
     private String serial;
     private String uptime;
     private long uptimeVal;
     private String boardName;
-    private int tempCpum;
-    private int tempSw;
-    private int tempCpub;
-    private int fanRpm;
```

Comment:
I would suggest to keep tempCpum, tempSw, tempCpub and fanRpm in case we have a user with an old firmware ?
This was kept for compatibility reason.
But that is true that I don't if this is really possible to keep an old firmware.
To be discussed but maybe we can suppress them.

Reply:
They are dynamically discovered depending upon existing sensors on the Server (not the same for Revolution and Delta). So they still exist in the "sensors" channel group.